plugins/chainlink.prebuilt.Dockerfile

```diff
@@ -0,0 +1,16 @@
+##
+# Takes Chainlink core as a base image and layers in private plugins.
+##
+ARG BASE_IMAGE=public.ecr.aws/chainlink/chainlink:v2.23.0-plugins
+
+##
+# Final image
+##
+FROM ${BASE_IMAGE} AS final
+# This directory should contain a bin/ subdir with the plugins and an optional lib/ subdir with shared libraries.
+ARG PKG_PATH=./build
+
+# Copy/override any (optional) additional shared libraries.
+COPY ${PKG_PATH}/li[b] /usr/lib/
```

Comment:
Are the square brackets some kind of well known hack?

Reply:
Yeah, added a comment to clarify this in 4291268a3e.

Reply:
It's a "hack" to make lib COPY optional (don't fail if `${PKG_PATH}/lib` doesn't exist), suggested here: https://stackoverflow.com/a/76040555